src/components/atoms/BackButton/BackButton.scss

```diff
@@ -0,0 +1,35 @@
+@import "scss/constants.scss";
```

Comment:
[note] I didn't deeply review these styles. @mike-lvov May have further thoughts on some of the patterns with them.

Reply:
@mike-lvov Some input would be nice here. Please take a look at the `scss` file when you are available.